src/e2e/deploy.e2e.ts

```diff
@@ -0,0 +1,66 @@
+import { run } from "./utils/shell";
+import { DeployDocumentStoreCommand, DeployTokenRegistryCommand } from "../commands/deploy/deploy.types";
+import { isAddress } from "web3-utils";
+import { defaultRunParameters, EndStatus, owner } from "./utils/constants";
+import { generateDeployDocumentStoreCommand, generateDeployTokenRegistryCommand } from "./utils/commands";
+import { getSigner, retrieveTokenInfo, rolesCheck } from "./utils/contract-checks";
+import { checkTokenRegistrySuccess, defaultTokenRegistry } from "./utils/helpers";
+
+export const deployTokenRegistry = async (): Promise<void> => {
```

Comment:
can add a suffixes to indicate its a e2e test?
hard to discern whether its a method or e2e (this goes for all the e2e entry point functions)